components/shared/fonts/font_template.rs

```diff
@@ -244,6 +244,14 @@ impl FontTemplate {
             .copied()
             .for_each(&mut add_variation);
 
+        // Step 9. Font variations implied by the value of the font-optical-sizing property are applied.
```

Comment:
Step 9 was in the wrong place previously (steps are in reverse order because variations are listed in ascending order of priority)